### PR TITLE
Doc: Prepare for 13.0-beta2 release

### PR DESCRIPTION
**Pull request by 2TallTyler**
## Motivation / Problem / Description

There are a lot of new features since the 13.0-beta1 release. We discussed on Discord that we should probably do a beta2 release before the RC at the end of this month.

## Limitations

- The release date isn't included.
- This will need updating with any commits between now and release.

## Checklist for review

Some things are not automated, and forgotten often. This list is a reminder for the reviewers.
* The bug fix is important enough to be backported? (label: 'backport requested')
* This PR touches english.txt or translations? Check the [guidelines](https://github.com/OpenTTD/OpenTTD/blob/master/docs/eints.md)
* This PR affects the save game format? (label 'savegame upgrade')
* This PR affects the GS/AI API? (label 'needs review: Script API')
    * ai_changelog.hpp, gs_changelog.hpp need updating.
    * The compatibility wrappers (compat_*.nut) need updating.
* This PR affects the NewGRF API? (label 'needs review: NewGRF')
    * newgrf_debug_data.h may need updating.
    * [PR must be added to API tracker](https://wiki.openttd.org/en/Development/NewGRF/Specification%20Status)

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+Feature: Multi-track level crossings (#9931)
+Feature: Add cargo filter support to vehicle list. (#8308)
+Feature: Display power-to-weight ratio in ground vehicle details GUI
+Feature: Add buttons to toggle music in the Game Options menu (#9727)
+Feature: Automatic console command screenshot numbering with a filename ending in '#' (#9781)
+Change: Let AI developers edit non-editable AI/Game Script Parameters (#8895)
+Change: Allow building docks on clearable watered object tiles (#8514)
+Fix #10011: Incorrect infrastructure totals when overbuilding bay road stop (#10143)
+Fix #10023: Allow negative input in text fields when needed (#10112)
+Fix #10117: Object burst limit allowed one fewer object than the setting
```
Comment (James103) [inline changelog.txt]:
```suggestion
Fix #10117: Object burst limit allowed one fewer object than the setting (#10120)
```

Also, add the PR number, `#10123`, at the end of the line `Feature: Display power-to-weight ratio in ground vehicle details GUI` above.

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+Feature: Add cargo filter support to vehicle list. (#8308)
+Feature: Display power-to-weight ratio in ground vehicle details GUI (#10123)
+Feature: Add buttons to toggle music in the Game Options menu (#9727)
+Feature: Automatic console command screenshot numbering with a filename ending in '#' (#9781)
+Change: Let AI developers edit non-editable AI/Game Script Parameters (#8895)
+Change: Allow building docks on clearable watered object tiles (#8514)
+Fix #10011: Incorrect infrastructure totals when overbuilding bay road stop (#10143)
+Fix #10023: Allow negative input in text fields when needed (#10112)
+Fix #10117: Object burst limit allowed one fewer object than the setting (#10120)
+Fix #9908: Close company windows only after all its infrastructure is removed (#9912)
```
Comment (JGRennison) [inline changelog.txt]: The line for #9908/#9912 is fairly mystifying unless you refer back to the issue which is being fixed.
Might it be worth re-phrasing it in terms of the problem which was fixed (i.e. "Fix crash which could occur when a company was deleted when a depot window was open" or similar)?

File: changelog.txt [hunk 3]
```diff
@@ -1,3 +1,26 @@
...
+Feature: Hotkey to honk a vehicle's horn (#10110)
+Feature: Add cargo filter support to vehicle list (#8308)
+Feature: Display power-to-weight ratio in ground vehicle details GUI (#10123)
+Feature: Add buttons to toggle music in the Game Options menu (#9727)
+Feature: Improved local authority action window (#9928)
+Feature: Automatic console command screenshot numbering with a filename ending in '#' (#9781)
+Feature: Allow AI/GS to be fully modified in scenario editor (#10152)
+Change: Let AI developers edit non-editable AI/Game Script Parameters (#8895)
+Change: Allow building docks on clearable watered object tiles (#8514)
+Change: Minor improvements to the new Finance GUI
```
Comment (James103) [inline changelog.txt]:
```suggestion
Change: Minor improvements to the new Finance GUI (#10168)
```

File: changelog.txt [hunk 4]
```diff
@@ -1,3 +1,26 @@
...
+Fix #10154: Network game desync related to setting a random company face (#10157))
+Fix #10155: Network games not syncing company settings properly (#10158
```
Comment (James103) [inline changelog.txt]:
```suggestion
Fix #10155: Network games not syncing company settings properly (#10158)
```

**Comment by 2TallTyler** [comment]
Note: Now ordered by Feature>Add>Change>Bug, then by PR order.

**Review by michicc — APPROVED** [review]

**Pull request merged**